themes/default/layouts/page/pricing.html

```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
-                    <span>Team members</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>3</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>3 – 25</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>10+</span>
-                </div>
-            </div>
+        <ul id="faq-headers" class="hidden lg:flex list-none mt-10">
+            <li class="px-10 py-3 mr-3 border-b-4 border-solid border-blue-600" data-faq-label="pricing">
+                <h4><a href="#faq-pricing" class="text-black">Credits and Pricing</a></h4>
+            </li>
+            <li class="px-10 py-3 mr-3" data-faq-label="product">
+                <h4><a href="#faq-product" class="text-gray-500">Product</a></h4>
+            </li>
+            <li class="px-10 py-3" data-faq-label="billing">
+                <h4><a href="#faq-billing" class="text-gray-500">Billing and Support</a></h4>
+            </li>
+        </ul>
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span><a class="link" href="{{ relref . "/docs/intro/console/organizations" }}">Organizations</a></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>1</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>1</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
-                </div>
-            </div>
+        <p class="text-2xl font-bold lg:hidden mt-12 pl-5">Credits and Pricing</p>
+        <ul class="list-none p-0 mx-5 xl:ml-12" data-faq-type="faq-pricing">
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "What is a Pulumi Credit?" "large_header" true) }}
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span><a class="link" href="{{ relref . "/docs/intro/console/project-and-stack-management" }}">Project stacks</a></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Up to 20</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span>Unlimited</span>
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                        A Pulumi Credit represents managing 1 resource for 1 hour. 1 resource represents a
```

Comment:
I get the urge to want to use numbers in these kinds of situations, but it's grammatically incorrect -- in prose, anything under 10 should be spelled out. (I'll only comment on this one instance.)
https://www.grammarly.com/blog/when-to-spell-out-numbers/

Reply:
@gthuang @joeduffy thoughts on this?

Reply:
Agree with the feedback.